lib/ims-hof-values-map.json

```diff
@@ -301,8 +301,8 @@
              "IMS": "Other"
             },
             {
-             "HOF": "gender-unknown",
-             "IMS": "DontKnow"
+             "HOF": "prefer-not-to-say",
+             "IMS": "Prefer not to say"
```

Comment:
You have to check the correct value for the IMS key with the business as it comes from them we don't create it ourselves

Reply:
Informed Katherine and Sabrina and currently  it is "DontKnow" in IMS